src/modules/admin/components/projectConfig/ProjectConfigPage.tsx

```diff
@@ -0,0 +1,42 @@
+import AddIcon from '@mui/icons-material/Add';
+import { Button, Paper } from '@mui/material';
+import { useState } from 'react';
+
+import ProjectConfigDialog from './ProjectConfigDialog';
+import ProjectConfigTable from './ProjectConfigTable';
+import PageTitle from '@/components/layout/PageTitle';
+import { ProjectConfigFieldsFragment } from '@/types/gqlTypes';
+
+const ProjectConfigPage = () => {
+  const [selected, setSelected] = useState<ProjectConfigFieldsFragment | null>(
+    null
+  );
+  return (
+    <>
+      <PageTitle
+        title='Project Configs'
+        actions={
+          <Button
+            onClick={() => setSelected({} as ProjectConfigFieldsFragment)}
```

Comment:
Hrm no I think this feels off too. Nothing is really "selected" here, its just that we want the dialog to open. I think we are missing state to track whether the dialog is open or not.

```
const [open, setOpen] = useState(false)

...
onClick={() => setOpen(true)


```

You will just need to pass `open` and `onClose` to the dialog properly.